x-pack/plugins/uptime/public/components/functional/monitor_list.tsx

```diff
@@ -137,19 +137,6 @@ export const MonitorListComponent = ({
                 </EuiLink>
               ),
           },
-          {
-            field: 'ping.url.full',
-            name: i18n.translate('xpack.uptime.monitorList.urlColumnLabel', {
-              defaultMessage: 'URL',
-            }),
-            render: (url: string, monitor: LatestMonitor) => (
-              <div>
-                <EuiLink href={url} target="_blank" color="text">
-                  {url} <EuiIcon size="s" type="popout" color="subdued" />
-                </EuiLink>
-              </div>
-            ),
-          },
```

Comment:
Just making sure you realize that the one you're removing isn't exactly identical to the one with the same "field" name that's in the code:
```js
{
  field: 'ping.url.full',
  name: i18n.translate('xpack.uptime.monitorList.urlColumnLabel', {
    defaultMessage: 'URL',
  }),
  render: (url: string, monitor: LatestMonitor) => (
    <div>
      <EuiLink href={url} target="_blank" color="text">
        {url} <EuiIcon size="s" type="popout" color="subdued" />
      </EuiLink>
      {monitor.ping && monitor.ping.monitor && monitor.ping.monitor.ip ? (
        <EuiText size="xs" color="subdued">
          {monitor.ping.monitor.ip}
        </EuiText>
      ) : null}
    </div>
  ),
}
```

Reply:
Just want to be sure you're removing the correct one before I give the :+1: :D